gr-utils/modtool/support/_gr_modtool

```diff
@@ -0,0 +1,25 @@
+#compdef gr_modtool
+#autoload
+
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright 2021 Marcus Müller
+# This module offers tab-completion for ZSH
+# It's rudimentary.
+
+local -a subcmds
+subcmds=(
+    'add:Adds a block to the out-of-tree module.'
```

Comment:
Zsh convention is to prefer imperative mood form for descriptions: "add a block to the out-of-tree module". You have that for all the other descriptions ("disable" rather than "disables"). I tend to think that's good practice elsewhere - documentation and git commit messages - which you might want to consider if you copied this from documentation. It is shorter and more natural in the absence of an explicit subject for a sentence.

We also don't typically capitalize the first word or add a full-stop.

Of course, zsh conventions are really not something you're in any way obliged to follow given that this is going to the gnuradio project. We also use only two spaces for indentation in completions.

Reply:
Agreed, this was just me being lazy (I regexed this out of `gr_modtool --help`). These *should* be imperatives, for consistency with other tooling. I can fix this here (and upstream, too)